src/components/PaginationItem/PaginationItem.test.tsx

```diff
@@ -1,32 +1,43 @@
-import { shallow } from "enzyme";
 import React from "react";
+import { render } from "@testing-library/react";
 
 import PaginationItem from "./PaginationItem";
 
 describe("<PaginationItem />", () => {
   // snapshot tests
   it("renders and matches the snapshot", () => {
-    const component = shallow(
+    const { container } = render(
       <PaginationItem number={1} onClick={jest.fn()} />
     );
 
-    expect(component).toMatchSnapshot();
+    expect(container).toMatchSnapshot();
   });
 
   // unit tests
   it("displays the page number", () => {
-    const component = shallow(
-      <PaginationItem number={1} onClick={jest.fn()} />
+    const { container } = render(
+      <PaginationItem number={1} onClick={jest.fn()} isActive />
     );
 
-    expect(component.find("button").text()).toEqual("1");
+    expect(container.querySelector("button")).toHaveTextContent("1");
```

Comment:
It's usually best to use testing library queries and screen, and not container methods directly.
```suggestion
    expect(screen.getByRole("button", { name: "1" })).toBeInTheDocument();
```